main/src/library/File.flix

```diff
@@ -59,4 +165,196 @@ namespace File {
         else
             readAllHelper(r, line :: acc)
 
+
+    ///
+    /// Returns a string containing the given file `f`.
+    ///
+    pub def read(file: String): Result[String, String] & Impure =
+        try {
+            import new java.io.File(String) as newFile;
+            import java.io.File.toPath();
+            import java.nio.file.Files:newBufferedReader(##java.nio.file.Path);
+            let javaFile = newFile(file);
+            let javaPath = toPath(javaFile);
+            let reader = newBufferedReader(javaPath);
+            Ok(readHelper(readAll(reader)))
+        } catch {
+            case ex: ##java.io.IOException =>
+                import java.lang.Throwable.getMessage();
+                Err(getMessage(ex))
+        }
+
+
+    ///
+    /// Helper function for read.
+    /// Converts a list of strings to a string with linebreaks between every string in `list`.
+    ///
+    def readHelper(list: List[String]): String & Pure =
+        List.foldLeft((x, y) -> x + "\n" + y, "", list)
```

Comment:
If we're reading a file that uses `\r\n`, we'll lose that during this conversion, which might be surprising in some corner cases. Is that worth worrying about?

Reply:
yeah I think it would be more expected to use [File.readString(_)](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html#readString(java.nio.file.Path)) or [File.readString(_, _)](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html#readString(java.nio.file.Path,java.nio.charset.Charset))